repairnator/kubernetes-support/queue-for-buildids/README.md

```diff
@@ -0,0 +1,18 @@
+# Introduction
```

Comment:
avoid multiple doc file, merge with the main README

Reply:
I have removed the readme in 'queue-for-buildids'